python-lib/ml_assisted_labeling_to_computer_vision/object_detection.py

```diff
@@ -0,0 +1,36 @@
+import json
+import logging
+import math
+from json import JSONDecodeError
+
+logger = logging.getLogger(__name__)
+
+
+def format_labeling_plugin_annotations(image_annotations):
+    """ Format labeling annotations to be compatible with dss object detection (deephub)
+        :param image_annotations: json serialized with labeling format:
+            '[{"top":118, "left":527,"width":174, "height":94, "label":"fish"}]'
+        :return: json serialized with deephub format:
+            '[{"bbox": [527, 118, 174, 94], "category":"fish"}]'
+    """
+
+    deephub_image_annotations = []
+    # if images were skipped during labeling plugin, their annotations is nan:
+    try:
+        if isinstance(image_annotations, str):
+            image_annotations = json.loads(image_annotations)
+        elif isinstance(image_annotations, float) and math.isnan(image_annotations):
+            logger.warning("Image was not labelled, skipping")
+            image_annotations = []
+        else:
+            raise ValueError()
```

Comment:
Add a message here specifying the problem
```suggestion
            raise ValueError("Unsupported type: {}".format(type(image_annotations)))
```